src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,91 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * Functions related to interacting with the user.
+ */
+public class Ui {
+    private static Scanner sc = new Scanner(System.in);
+
+    // For checking whether to continue running Duke, i.e. exit command has not been called.
```

Comment:
Would it be a good idea to put the comments for class members as JavaDocs. For example

/** Number of connections to this database */
private int connectionCount;